util/audio_generate_dac_lut.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
```

Comment:
I think this needs a copyright header, right?

Reply:
@JohSchneider Just wanted to make sure, was this written by you?